faiss_vector_posting.go

```diff
@@ -335,9 +335,8 @@ func (sb *SegmentBase) InterpretVectorIndex(field string, except *roaring.Bitmap
 				return rv, nil
 			},
 			close: func() {
-				if vecIndex != nil {
-					vecIndex.Close()
-				}
+				// skipping the closing for now because the index is cached
+				// todo: subjected to change based on a optimization type flag
```

Comment:
Let's not make this an optimization thing - this'll be the new default going forward - applicable to _all_ optimizations.